src/applications/pensions/components/NoFormPage.jsx

```diff
@@ -95,6 +95,30 @@ export const NoFormPage = () => {
                 <br />
               </p>
             </div>
+            <va-link
+              href="/pension/survivors-pension/"
+              text="Learn about how to apply in person, or with the help of a VSO"
+            />
+            <h2>Your intent to file date</h2>
+            <div className="vads-u-background-color--gray-lightest vads-u-padding-y--2 vads-u-padding-x--3">
+              <h3
+                className="vads-u-margin-y--0 vads-u-padding-bottom--0p5 vads-u-border-bottom--2px vads-u-border-color--gray-light"
+                style={{
```

Comment:
Does this class `vads-u-display--inline` allow you to avoid inline styles?

Reply:
Yes it does thanks for that. Fixed